packages/snap-preact/components/src/components/Atoms/Icon/Icon.tsx

```diff
@@ -112,4 +112,5 @@ export type IconNames =
 	| 'close'
 	| 'option'
 	| 'expand'
-	| 'collapse';
+	| 'collapse'
+	| 'clear-all';
```

Comment:
Seems a little too generic - maybe it should be named "reset-facet" - as this name could apply to FilterSummary buttons...